pkg/skaffold/deploy/helm/helm_windows_test.go

```diff
@@ -0,0 +1,108 @@
+// +build windows
+
+/*
+Copyright 2020 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helm
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/mitchellh/go-homedir"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	schemautil "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/warnings"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestSetFilesFlagInDeploy(t *testing.T) {
```

Comment:
It would seem better to extract `sanitizeFilePath()` as a library function into `pkg/skaffold/util` and just have specific tests around that.

Maybe we can make the file-sanitizer pluggable so that we can remove Windows-isms from these tests?

Reply:
I actually think this is very specific to the way helm [handles](https://github.com/helm/helm/blob/d55c53df4e394fb62b0514a09c57bce235dd7877/pkg/cli/values/options.go#L75) its flags. We otherwise use `exec.Cmd` everywhere else and the windows [`syscall`](https://github.com/golang/go/blob/6951da56b0ae2cd4250fc1b0350d090aed633ac1/src/syscall/exec_windows.go#L27) package implements its own sanitizing for command args. So I don't think we need to move this to a common util.